app/models/user.rb

```diff
@@ -65,9 +65,9 @@ def self.eligible_voters
   end
 
   def self.leader_senior_voters
-    team_leaders = TeamMember.current.in_official_team.leader.map(&:user)
+    team_leaders = UserGroup.teams_committees.map(&:lead_user)
```

Comment:
Same question as above (feel free to resolve both by just answering one): Does this account for **current** Team Leaders only?